dev/com.ibm.ws.springboot.fat20_fat/fat/src/com/ibm/ws/springboot/support/fat/AbstractSpringTests.java

```diff
@@ -54,6 +55,9 @@ public abstract class AbstractSpringTests extends TestContainerSuite {
     @Rule
     public TestName testName = new TestName();
 
+    @Rule
+    public static final SkipJavaSemeruWithFipsEnabled skipJavaSemeruWithFipsEnabled = new SkipJavaSemeruWithFipsEnabled("SpringBootTests");
+
```

Comment:
What is the impact of this `@Rule` on the whole bucket?  Does it cause the complete bucket to be skipped?  Or does it only allow the `@SkipJavaSemeruWithFipsEnabledRule` to be used by various test methods that we want to skip?